views/blog/index.ejs

```diff
@@ -1,3 +1,111 @@
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
+<p>It will list the latest 5 blogs from most recent to oldest</p>
+<h1>Blogs</h1>
+<p>This is the home page</p>
```

Comment:
Hey Noel, why was this h1 and p tag repeated so many times? Is this supposed to represent something?